templates/default/datadog.conf.erb

```diff
@@ -183,6 +183,11 @@ end
 apm_enabled: <%= node['datadog']['enable_trace_agent'] %>
 <% end -%>
 
+[trace.config]
+<% unless node['datadog']['trace_env'].nil? -%>
+env = <%= node['datadog']['trace_env'] %>
```

Comment:
Oh yeah totally missed that. I copied the line from documentation initially, so that could be changed as well